lib/sentry/client.ex

```diff
@@ -117,7 +117,7 @@ defmodule Sentry.Client do
 
     result_type = Keyword.get_lazy(opts, :result, &Config.send_result/0)
     client = Keyword.get_lazy(opts, :client, &Config.client/0)
-    sample_rate = Keyword.get_lazy(opts, :sample_rate, &Config.sample_rate/0)
+    sample_rate = Keyword.get_lazy(opts, :sample_rate, &Config.traces_sample_rate/0)
```

Comment:
the errors `sample_rate` and transactions `traces_sample_rate` are two different options and should be kept separate and used for the relevant sampling decision, so please don't replace the original one.

Reply:
This comment is not correct. It's not replacing the original one, it's switching from `sample_rate` to `traces_sample_rate` in this function, which reports *transactions*. We didn't have `traces_sample_rate` before, so we used `sample_rate` as a stopgap.

Reply:
I'm saying we need both